operator/src/main/helm/coherence-operator/dashboards/coherence-dashboard-main.json

```diff
@@ -410,7 +406,7 @@
       "tableColumn": "",
       "targets": [
         {
-          "expr": "count(avg(coherence_service_partitions_endangered{cluster=~\"$ClusterName\",tag_type=~\"DistributedCache|FederatedCache\"}) by(name) > 0)",
+          "expr": "count(coherence_partition_assignment_HA_status{cluster=~\"$ClusterName\"} == 1)",
```

Comment:
does the above need a "by (coherence_serivce)" added to it like the HA_status reference later on in this file.

Also, not sure this should be a count, there should be only one HA_status per cluster,coherence_service metrics labels.

Reply:
The above expression is to show the total count of services that are in endangered status. I will make this one <= 1 as well. The by (coherence_service), won't be needed as this is a total.